SecurityPkg/VariableAuthenticated/SecureBootConfigDxe/SecureBootConfigImpl.c

```diff
@@ -4651,8 +4690,49 @@ SecureBootCallback (
       // When entering SecureBoot OPTION Form
       // always close opened file & free resource
       //
-      if ((QuestionId == KEY_SECURE_BOOT_PK_OPTION) ||
-          (QuestionId == KEY_SECURE_BOOT_KEK_OPTION) ||
+      if ((QuestionId == KEY_SECURE_BOOT_PK_OPTION)){
+        CloseEnrolledFile (Private->FileContext);
+
+        GetVariable2 (EFI_PLATFORM_KEY_NAME, &gEfiGlobalVariableGuid, (VOID **)&PkList, NULL);
+        if (PkList != NULL) {
+          // Extract the PK certificate from the list
+          PkCert = (EFI_SIGNATURE_DATA *)((UINT8 *)PkList + sizeof (EFI_SIGNATURE_LIST) + PkList->SignatureHeaderSize);
+          UINTN cnSize = 0;
+          X509GetCommonName(
+            (UINT8*)PkCert->SignatureData,
+            (UINTN)PkList->SignatureSize,
+            NULL,
+            &cnSize);
+
+          PkCN8 = AllocateZeroPool(cnSize*sizeof(CHAR8));
+          if (PkCN8 == NULL) {
+            return EFI_OUT_OF_RESOURCES;
+          }
+          PkCN16 = AllocateZeroPool(cnSize*sizeof(CHAR16));
+          if (PkCN16 == NULL) {
+            return EFI_OUT_OF_RESOURCES;
+          }
+
+          RETURN_STATUS RetVal = X509GetCommonName(
+            (UINT8*)PkCert->SignatureData,
+            (UINTN)PkList->SignatureSize,
+            PkCN8,
+            &cnSize);
+
+          if (RetVal == RETURN_SUCCESS){
+            AsciiStrToUnicodeStrS(PkCN8, PkCN16, cnSize);
+            HiiSetString (Private->HiiHandle, STRING_TOKEN (STR_PK_NAME), PkCN16, NULL);
+          } else {
+            HiiSetString (Private->HiiHandle, STRING_TOKEN (STR_PK_NAME), L"*NO COMMON NAME*", NULL);
+          }

```

Comment:
```suggestion
          if (!EFI_ERROR(RetVal)) {
            AsciiStrToUnicodeStrS(PkCN8, PkCN16, cnSize);
            HiiSetString (Private->HiiHandle, STRING_TOKEN (STR_PK_NAME), PkCN16, NULL);
          } else {
            HiiSetString (Private->HiiHandle, STRING_TOKEN (STR_PK_NAME), L"*NO COMMON NAME*", NULL);
          }

```

Reply:
THe code style is inconsistent in many plases. Many spaces are missing before `{` or `(`

Reply:
[Fixed](https://github.com/Dasharo/edk2/pull/149/files#diff-49fdb815b38763cce7b30e2aea02c17111bc51206950ff1d21e0073ebd0cb1d3R2685) here and many other places